docs/src/starknet/account.md

```diff
@@ -64,61 +64,28 @@ command: account deploy
 transaction_hash: 0x20b20896ce63371ef015d66b4dd89bf18c5510a840b4a85a43a983caa6e2579
 
 To see invocation details, visit:
-transaction: https://starkscan.co/tx/0x20b20896ce...
-    ```
-
-    Note that you don't have to pass `url`, `accounts-file` and `network` parameters if `add-profile` flag
-    was set in the `account create` command. Just pass `profile` argument with the account name.
+transaction: https://starkscan.co/search/0x20b20896ce...
+```
 
-    For a detailed CLI description, see [account deploy command reference](../appendix/sncast/account/deploy.md).
+For a detailed CLI description, see [account deploy command reference](../appendix/sncast/account/deploy.md).
 
 > 💡 **Info**
 > You can also choose to pay in Ether by setting `--fee-token` to `eth`.
 
-### `account create` With Salt Argument
+## Additional features
 
-Salt will not be randomly generated if it's specified with `--salt`.
+### [`account list`](../appendix/sncast/account/list.md)
+List all accounts saved in `accounts file`, grouped based on the networks they are defined on.
 
 ```shell
-$ sncast \
-    account create \
-    --name some-name \
-    --salt 0x1
-
-Account successfully created. Prefund generated address with at least 432300000000 tokens. It is good to send more in the case of higher demand, max_fee * 2 = 864600000000
-command: account create
-max_fee: 0x64a7168300
-address: 0x7a949e83b243068d0cbedd8d5b8b32fafea66c54de23c40e68b126b5c845b61
-
-To see account creation details, visit:
-account: https://starkscan.co/tx/0x7ad0d6e449...
+$ sncast account list
 ```
 
-### `account delete`
-
-Delete an account from `accounts-file` and its associated Scarb profile.
-
 ```shell
-$ sncast \
-    --accounts-file my-account-file.json \
-    account delete \
-    --name some-name \
-    --network alpha-sepolia
-
-Do you want to remove account some-name from network alpha-sepolia? (Y/n)
-Y
-command: account delete
-result: Account successfully removed
+$ sncast --accounts-file my-account-file.json account list
```

Comment:
I'd group output with the `sncast account list` and just mention below that you can also lists accounts in specific file by providing `--accounts-file`. But I wouldn't show the whole output for that.